src/votekit/metrics.py

```diff
@@ -0,0 +1,95 @@
+from votekit.cvr_loaders import rank_column_csv
+from votekit.election_types import STV, fractional_transfer
+
+
+fpath = "/Users/emariedelanuez/VoteKit/src/votekit/mn_clean_ballots.csv"
+
+
+def loadin(fpath):
+    """
+    The goal of this file is to output the distance
+    between the rankings produced by the STV
+    results of a race and an unordered
+    list of candidates.
+    """
+    data = rank_column_csv(fpath)
+    return data
+
+
+data = loadin(fpath)
```

Comment:
global variables like this should be removed (especially ones w/ names that mask parameters like `data`)